asab/library/providers/libsreg.py

```diff
@@ -28,26 +29,50 @@ class LibsRegLibraryProvider(FileSystemLibraryProvider):
 
 	It provides an option to specify more servers for more reliable content delivery.
 
-	Example of the configuration:
+	Configuration example:
 
 	```ini
 	[library]
 	providers=
-	...
-	libsreg+https://libsreg1.example.com,libsreg2.example.com/my-library
-	...
+		...
+		libsreg+https://libsreg1.example.com,libsreg2.example.com/my-library
+		...
 	```
 
+	Specify the library version in the URL fragment (after `#`):
+	```ini
+	[library]
+	providers=
+		...
+		libsreg+https://libsreg1.example.com,libsreg2.example.com/my-library#v25.14.01
+		...
+	```
+
+	Specify the pull interval after `@pull:`:
+	```ini
+	[library]
+	providers=
+		...
+		libsreg+https://libsreg1.example.com,libsreg2.example.com/my-library@pull:24h
+		...
+	```
 	"""
 
+
 	def __init__(self, library, path, layer):
 
+		if "@pull:" in path:
```

Comment:
You should extract this from parsed URL ... and it should be a "fragment" `#` ... to NOT send that to a remove server.